### PR TITLE
New Signals: Attachment and Sender-based

### PR DESCRIPTION
**Pull request by bfilar**

### CODE REVIEW

File: signals/attachments/attachment_content_type_delivery_status.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: "Attachment: Content Type Message Delivery Status Notification"
```
Comment (aidenmitchell) [inline signals/attachments/attachment_content_type_delivery_status.yml:1]:
```suggestion
name: "Attachment: Message Delivery Status Notification Content Type"
```
just to have it the same way as the "EML Content Type" signal!

File: signals/sender/sender_display_name_suspicious_docusign.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: "Contains Suspicious DocuSign Display Names"
```
Comment (aidenmitchell) [inline signals/sender/sender_display_name_suspicious_docusign.yml:1]:
```suggestion
name: "Sender: Contains Suspicious DocuSign Display Names"
```

**Comment by aidenmitchell** [comment]
https://github.com/sublime-security/sublime-rules/pull/1294/files#diff-c59f897907f4d1eaec1257c651fdde88658fc2fbe01be4c1d74c4702a0736bdbL1
also, this seems to have an extra space at the end of the name?

**Comment by bfilar** [comment]
good catch! Changes made!

**Review by aidenmitchell — APPROVED** [review]

**Pull request merged**